drc.map.txt

```diff
@@ -1,5 +1,7 @@
 [developer.rackspace.com]
 / https://github.com/rackerlabs/docs-developer-blog/
 /docs/ https://github.com/rackerlabs/docs-quickstart/
+/docs/user-guides/infrastructure https://github.com/rackerlabs/docs-core-infra-user-guide/
 
 /favicon.ico >>https://8d8dcdd952aa2708c2ff-519cda130c91226e76017ae910bdb276.ssl.cf1.rackcdn.com/favicon-aa186bee158ecea4c9b6a98e2ceec3141b6b7d8d94dcb71731ba112e2b17b1db.ico
+/assets/ >>http://devassets:8080/
```

Comment:
This line probably shouldn't be here when we merge - it'll mess up the asset CDN hosting.